.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: node_js
+node_js:
+ - '10.1.0'
```

Comment:
Change this to `node` so it uses whatever is current vs. pinning to a random version.